README.md

```diff
@@ -1,46 +1,90 @@
-# Getting Started with Create React App
+![Imgur](https://i.imgur.com/QQWxbJs.png)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## What is it? :rooster:
+Todooster is a fullstack app that uses React for the frontend, express for the server, and MySQL for the database.
 
-## Available Scripts
+### See it in action!
+![Todooster App Gif](https://media.giphy.com/media/C3Wwc75PYIi6psYbPO/giphy.gif)
 
-In the project directory, you can run:
+## What's the user experience?
+Todooster is designed to be a simple todo app that allows a user to jot down tasks to complete and remove them when completed.
 
-### `yarn start`
+## Functionality for user
+- Add new tasks to the list
+- Cross off completed tasks
+- Delete completed tasks
+- Login and have the state for the list be saved
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## What technologies are being used?
+- [Node.js](https://nodejs.org/en/) v12.18.3
+- [npm](https://www.npmjs.com/) v7.18.1
+- [React w/ Typescript](https://create-react-app.dev/)
+- [Tailwind CSS w/ Craco](https://tailwindcss.com/)
+- [Jest](https://jestjs.io/) (for testing)
+- [React Icons](https://react-icons.github.io/react-icons/)
+- [MySQL](https://www.mysql.com/) database
+- [Goose](https://github.com/pressly/goose) for migrations
+- [Express](https://expressjs.com/)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+# Setup :hatching_chick:
+1. Download the .zip file or clone the repo using the command line tool:
+```
+gh repo clone chris-chen-creates/todooster
+```
+2. Install the dependencies using:
```

Comment:
Maybe one more comment about migrating the database

Reply:
donezo!